documentation/3.4.0/setup/native/project-with-survey/ubuntu/README.md

```
@@ -0,0 +1,385 @@
+
+        3. Enable Citus and set distribution columns for `user` database by running the `citus_setup.sh`with the following arguments.
+            ```
+             ./citus_setup.sh user postgres://postgres:postgres@localhost:9700/users
+            ```
+
+9.  **Update Cloud Credentials for Project Service**
+
+    To enable full functionality—including certificate generation, attachment uploads, and report storage—you must configure cloud credentials in the environment files for both services.
+
+    A. Project Service Configuration Path:
+    	```./ELEVATE-Project/project-service/.env
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 5

(MD010, no-hard-tabs)